lib/ansible/modules/remote_management/foreman/foreman_global_parameter.py

```diff
@@ -0,0 +1,198 @@
+author: "Matthias Dellweg (@mdellweg) and Bernhard Hopfenmueller(@Fobhep)"
+requirements:
+    - "nailgun >= 0.29.0"
+    - "python >= 2.6"
+options:
+    server_url:
+        description:
+            - URL of Foreman server
+        required: true
+    username:
+        description:
+            - Username on Foreman server
+        required: true
+    password:
+        description:
+            - Password for user accessing Foreman server
+        required: true
+    verify_ssl:
+        description:
+            - Verify SSL of the Foreman server
+        required: false
+    name:
+        description:
+            - Name of the Global Parameter
+        required: true
+    value:
+        description:
+            - Value of the Global Parameter
+        required: false
+    state:
+        description:
+            - State of the Global Parameter
+        required: false
+'''
+
+EXAMPLES = '''
+- name: "Create a Global Parameter"
+  local_action:
+      module: foreman_global_parameter
+      username: "admin"
+      password: "admin"
+      server_url: "https://fakeserver.com"
+      name: "TheAnswer"
+      value: "42"
+      state: present
+
+- name: "Delete a Global Parameter"
+  local_action:
+      module: foreman_global_parameter
+      username: "admin"
+      password: "admin"
+      server_url: "https://fakeserver.com"
+      name: "TheAnswer"
+      state: absent
+'''
+
+RETURN = '''# '''
+
+try:
+    from nailgun.config import ServerConfig
+    import nailgun.entity_mixins
+    import nailgun.entities
+    HAS_NAILGUN_PACKAGE = True
+except:
+    HAS_NAILGUN_PACKAGE = False
+
+from ansible.module_utils.basic import AnsibleModule
+
+'''CommonParameter needs to inherit functionalities
+   for searching, updating and deleting'''
+
+
+class CommonParameter(
+    nailgun.entities.CommonParameter,
+    nailgun.entity_mixins.Entity,
+    nailgun.entity_mixins.EntityCreateMixin,
+    nailgun.entity_mixins.EntityDeleteMixin,
+    nailgun.entity_mixins.EntityReadMixin,
+    nailgun.entity_mixins.EntitySearchMixin,
+    nailgun.entity_mixins.EntityUpdateMixin
+):
+
+    pass
+
+
+def create_CommonParameter(name, value):
+    CommonParameter(name=name, value=value).create()
+
+
+def update_CommonParameter(name, value):
+    CommonParameter(name=name, value=value).update()
+
+
+def create_Server(server_url, auth, verify_ssl):
+    ServerConfig(
+        url=server_url,
+        auth=auth,
+        verify=verify_ssl,
+    ).save()
+
+
+def find_Entity(entity, **kwargs):
+    instance = entity(**kwargs)
+    return instance.search(
+        set(),
+        {'search': '%s="%s"' % (key, kwargs[key]) for key in kwargs}
+    )
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            server_url=dict(required=True),
+            username=dict(required=True, no_log=True),
+            password=dict(required=True, no_log=True),
+            verify_ssl=dict(required=False, type='bool', default=False),
```

Comment:
Please change the default to true (assuming this is what other Forman modules do)